css/main.css

```diff
@@ -1,3 +1,209 @@
+header{
+  background-color: white;
+}
+.bulletpoint{
+width: 20px;
+height: 20px;
+}
+.menue{
+  height: 20px;
+  width: 20px;
+}
+.topnav {
+  background-color:white;
+  overflow: hidden;
+}
+.topnav a {
+  display:flex;
+  flex-direction: column;
+  justify-content:right;
+  color: black;
+  padding: 14px 16px;
+  text-decoration: none;
+  font-size: 17px;
+}
+.topnav a.active {
+  color: blue;
+}
+.topnav .icon {
+  display: none;
+}
+@media screen and (min-width: 0px) {
+  .topnav a:not(:nth-child(0)), .dropdown .dropbtn {
+    display: none;
+  }
+  .topnav a.icon {
+    display: flex;
+    float: right;
+    justify-content: flex-end;
+    flex-direction: column;
+  }
+  .topnav.responsive {position: relative;}
+  .topnav.responsive a.icon {
+    position: absolute;
+    right: 0;
+    top: 0;
+  }
+  .topnav.responsive a {
+    display: block;
+    text-align: left;
+  }
+
+}
+@media screen and (min-width: 600px){
+.topnav.responsive a{
+  display: flex;
+  flex-direction: row;
+  justify-content: left;
+}
+  .topnav a:not(:nth-of-type(0)), .dropdown .dropbtn {
+    display: flex;
+    flex-direction: row;
+
+  }
+  .topnav a.icon {
+    display: none;
+  }
+  .topnav{
+    display: flex;
+    justify-content: space-evenly;
+    flex-direction: row;
+  }
+}
+.maintop{
+  background-color: white;
+  padding-bottom: 20%;
+}
+
+.menue1{
+  width: 35px;
+  height: 5px;
+  background-color: blue;
+  margin: 6px 0;
+}
+body{
+  background-color: wheat;
+}
+.backgroundcolortop{
+  background-color:white;
+  height: 200px;
+  width: 100%;
+}
+div.transbox {
+  margin-left: 60px;
+  margin-right: 20px;
+  background-color:antiquewhite;
+  opacity: 1;
+  box-shadow:20px 20px 50px grey;
+}
+
+div.transbox p {
+  margin: 5%;
+  font-weight: bold;
+  color: #000000;
+}
+
+.image{
+  border-radius: 50%;
+  height: auto;
+  display: block;
+  margin-left: auto;
+  margin-right: auto;
+  width: 50%;
+}
+.linebreak{
+  color: blue;
+}
+.projectsbutton{
+  background-color: antiquewhite;
+  border-radius: 20px;
+}
+.projectsbutton:hover{
+  background-color: blue;
+  color: white;
+}
+.resumebutton{
+  background-color: blue;
+  color: white;
+  border-radius: 20px;
+  border-color: blue;
+}
+.resumebutton:hover{
+  background-color: antiquewhite;
+  color: black;
+}
+.size{
+  padding-bottom: 20px;
+}
+.socialmediaicons{
+  padding-bottom: 20px;
+  padding-left: 11px;
+  padding-right: 11px;
+}
+.icons{
+  background-color: white;
+}
+.maintextposition{
+  padding-top: 30%;
+}
+body .footerbackgroundcolor{
+  background-color: white;
+}
+@media screen and (min-width:450px){
+  .maintextposition {
+    padding-top: 10%;
+}
+}
+@media screen and (min-width:775px){
+  .maintextposition {
+    padding-bottom: 35%;
+}
+  .main{
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+  }
+  .maintextposition{
+    padding-top: 0px;
+    padding-bottom: 70px;
+    box-shadow:20px 20px 50px grey;
+    background-color: white;
```

Comment:
This Hello section is going all the way to the right edge of the screen because it is not contained inside a `col-...` class
As mentioned in other comments, move the container up to the parent, to wrap the _whole_ block (profile image and text). And also add a `row` and `col-12` at the top level before you set the white background. See my suggested code for your resume page for inspiration.